subprojects/core/src/main/java/org/gradle/api/internal/tasks/AbstractTaskOutputPropertySpec.java

```diff
@@ -48,19 +47,18 @@ public TaskOutputFilePropertyBuilder optional(boolean optional) {
         return this;
     }
 
-    public SnapshotNormalizationStrategy getSnapshotNormalizationStrategy() {
-        return snapshotNormalizationStrategy;
+    public PathNormalizationStrategy getPathNormalizationStrategy() {
+        return OutputPathNormalizationStrategy.getInstance();
     }
 
     @Override
     public TaskOutputFilePropertyBuilder withPathSensitivity(PathSensitivity sensitivity) {
-        this.snapshotNormalizationStrategy = TaskFilePropertySnapshotNormalizationStrategy.valueOf(sensitivity);
-        return this;
+        throw new UnsupportedOperationException("Cannot change path sensitivity of output file property");
```

Comment:
I added the removal of `TaskFileProperty.withPathSensitivity` to the release notes.